src/Common/HashTable/HashTable.h

```diff
@@ -1036,14 +1036,16 @@ class HashTable :
         return const_cast<std::decay_t<decltype(*this)> *>(this)->find(x, hash_value);
     }
 
-    std::enable_if_t<Grower::performs_linear_probing_with_single_step, bool>
-    ALWAYS_INLINE erase(const Key & x)
+    template <typename Grower_ = Grower>
+    requires (Grower_::performs_linear_probing_with_single_step)
```

Comment:
Here it is better to leave previous implementation. With concepts it does not look obvious. Same for function below.

Reply:
I agree, and will provide a fix as requested.